portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
Maybe update this method name and variable names to be more indicative of what it is doing.